tracker-db/openshift/postgresql-build.tracker.param

```diff
@@ -0,0 +1,13 @@
+#=========================================================
```

Comment:
We don't use this kind of param files any more.

Reply:
removed